pkg/controller/operators/olm/apiservices.go

```diff
@@ -307,24 +307,25 @@ func (a *Operator) installOwnedAPIServiceRequirements(csv *v1alpha1.ClusterServi
 	// Assume the strategy is for a deployment
 	strategyDetailsDeployment, ok := strategy.(*v1alpha1.StrategyDetailsDeployment)
 	if !ok {
-		return nil, fmt.Errorf("unsupported InstallStrategy type")
+		return nil, nil, fmt.Errorf("unsupported InstallStrategy type")
 	}
 
 	// Return early if there are no owned APIServices
 	if len(csv.Spec.APIServiceDefinitions.Owned) == 0 {
-		return strategyDetailsDeployment, nil
+		return strategyDetailsDeployment, nil, nil
 	}
 
 	// Create the CA
 	expiration := time.Now().Add(DefaultCertValidFor)
 	ca, err := certs.GenerateCA(expiration, Organization)
 	if err != nil {
 		logger.Debug("failed to generate CA")
-		return nil, err
+		return nil, nil, err
 	}
 	rotateAt := expiration.Add(-1 * DefaultCertMinFresh)
 
 	apiDescs := csv.GetOwnedAPIServiceDescriptions()
+	deploymentNameToCAPEM := map[string][]byte{}
```

Comment:
Best to use the `make` built-in when initializing maps ie `deploymentNameToCAPEM := make(map[...])`

Reply:
Updated, but based on [this](https://stackoverflow.com/questions/31064688/which-is-the-nicer-way-to-initialize-a-map-in-golang) conversation from StackOverflow there is no performance gain.